lib/paginative/models/ordering_helpers.rb

```diff
@@ -3,20 +3,13 @@ module OrderingHelpers
     extend ActiveSupport::Concern
 
     included do
-      def self.sanitized_ordering(table_name, field, order)
-        if field.is_a? Array
-          return raise "Wrong number of sorting fields. Expected 2, got #{field.length}. If you want to sort by a singular field please pass field argument as a string rather than an array." unless field.length == 2
-
-          "#{table_name}.#{sanitize_column(field[0])} #{sanitize_column_direction(order)}, #{table_name}.#{sanitize_column(field[1])} #{sanitize_column_direction(order)}"
```

Comment:
Here you are calling `sanitize_column()` but it looks like it is no longer defined anywhere